docs/source/kale.loaddata.rst

```diff
@@ -102,6 +102,14 @@ kale.loaddata.video\_multi\_domain module
    :undoc-members:
    :show-inheritance:
 
+kale.loaddata.office\_access
+------------------------------------------
+
+.. automodule:: kale.loaddata.office_access
```

Comment:
Check the built doc: https://pykale--197.org.readthedocs.build/en/197/kale.loaddata.html#module-kale.loaddata.office_access 
Compare with https://pykale--197.org.readthedocs.build/en/197/kale.loaddata.html#module-kale.loaddata.cifar_access, https://pykale--197.org.readthedocs.build/en/197/kale.loaddata.html#module-kale.loaddata.digits_access, and https://pykale--197.org.readthedocs.build/en/197/kale.loaddata.html#module-kale.loaddata.video_access to see what docs you need to provide.